internals/logger/logger.go

```diff
@@ -120,7 +118,7 @@ func securityEvent(level string, event SecurityEvent, arg, description string) {
 		// Should never happen, and not much more we can do here.
 		return
 	}
-	logger.Notice(buf.String())
+	logger.Noticef("%s", buf.String())
```

Comment:
I believe you can use `buf.Bytes()` with the `%s` verb, which should avoid an allocation here.